development/github/pkg/client/client.go

```diff
@@ -1 +1,116 @@
 package client
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"net/http"
+
+	"github.com/google/go-github/v31/github"
+	"github.com/kyma-project/test-infra/development/types"
+	"golang.org/x/oauth2"
+	"gopkg.in/yaml.v2"
+)
+
+const (
+	SapToolsGithubURL  = "https://github.tools.sap/"
+	ProwGithubProxyURL = "http://ghproxy"
```

Comment:
I did it intentionally. ProwGithubProxyURL is not used yet. SapToolsGithubURL is there for use with NewSapToolClient, which creates client for this specific endpoint. This client provides specific methods valid for our instance. I would like to avoid adding another config parameter which must be delivered in a way specific to the place of running the code. I think this is the most convenient way for using this client.